src/main/resources/application.yml

```
@@ -55,3 +55,12 @@ logging:
 
 fcm:
   certification: firebase.json
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include: prometheus
+  endpoint:
+    prometheus:
+      enabled: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)